jwcrypto/jwk.py

```diff
@@ -451,13 +447,13 @@ def _generate_OKP(self, params):
             raise InvalidJWKValue('Must specify "crv" for OKP key generation')
         try:
             key = _OKP_CURVES_TABLE[params['crv']].privkey.generate()
-        except KeyError:
+        except KeyError as e:
             raise InvalidJWKValue('"%s" is not a supported curve for the '
-                                  'OKP key type' % params['crv'])
+                                  'OKP key type' % params['crv']) from e
         self._import_pyca_pri_okp(key, **params)
 
     def _okp_curve_from_pyca_key(self, key):
-        for name, val in iteritems(_OKP_CURVES_TABLE):
+        for name, val in iter(_OKP_CURVES_TABLE.items()):
```

Comment:
``items`` returns an iterator.

```suggestion
        for name, val in _OKP_CURVES_TABLE.items():
```